src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -40,7 +40,7 @@ public void DumpRuntimeInformationToConsole()
 
             Console.WriteLine($"### CURRENT DIRECTORY: {Environment.CurrentDirectory}");
 
-            string cgroupsLocation = Interop.cgroups.s_cgroupMemoryPath;
+            string cgroupsLocation = Interop.cgroups.s_cgroupMemoryLimitPath;
```

Comment:
Test output example on my system:

<details>
        <summary>Click to expand</summary>

```
  ### OS: Distro=Distro=fedora VersionId=30.0 Description=Linux 5.1.18-300.fc30.x86_64 #1 SMP Mon Jul 15 15:42:34 UTC 2019 Version=Unix 5.1.18.300 Arch=X64
  ### LIBC: Release=stable Version=2.29
  ### FRAMEWORK: Version=5.0.0 Description=.NET Core 5.0.0
  ### BINARIES: /home/omajid/devel/dotnet/corefx/artifacts/bin/testhost/netcoreapp-Linux-Debug-x64/shared/Microsoft.NETCore.App/5.0.0 (drive format ext4)
  ### TEMP PATH: /tmp/ (drive format tmpfs)
  ### CURRENT DIRECTORY: /home/omajid/devel/dotnet/corefx/artifacts/bin/System.Runtime.InteropServices.RuntimeInformation.Tests/netcoreapp-Unix-Debug
  ### CGROUPS MEMORY: /sys/fs/cgroup/user.slice/user-1000.slice/user@1000.service/gnome-terminal-server.service/memory.max
  ### ENVIRONMENT VARIABLES
<removed>
  ### PROCESS INFORMATION:
  ###   Architecture: X64
  ###   BasePriority: 0
  ###   HandleCount: 177
  ###   Id: 23194
  ###   MachineName: .
  ###   MainModule: System.Diagnostics.ProcessModule (dotnet)
  ###   MainWindowHandle: 0
  ###   MainWindowTitle:
  ###   MaxWorkingSet: 9223372036854775807
  ###   MinWorkingSet: 0
  ###   NonpagedSystemMemorySize: 0
  ###   NonpagedSystemMemorySize64: 0
  ###   PagedMemorySize: 0
  ###   PagedMemorySize64: 0
  ###   PagedSystemMemorySize: 0
  ###   PagedSystemMemorySize64: 0
  ###   PeakPagedMemorySize: 0
  ###   PeakPagedMemorySize64: 0
  ###   PeakVirtualMemorySize: 0
  ###   PeakVirtualMemorySize64: 0
  ###   PeakWorkingSet: 0
  ###   PeakWorkingSet64: 0
  ###   PriorityBoostEnabled: False
  ###   PriorityClass: Normal
  ###   PrivateMemorySize: 0
  ###   PrivateMemorySize64: 0
  ###   PrivilegedProcessorTime: 00:00:00.0300000
  ###   ProcessName: dotnet
  ###   ProcessorAffinity: 255
  ###   Responding: True
  ###   SessionId: 10889
  ###   StartTime: 7/22/2019 10:07:42 PM
  ###   TotalProcessorTime: 00:00:00.9200000
  ###   UserProcessorTime: 00:00:00.8900000
  ###   VirtualMemorySize: -471937024
  ###   VirtualMemorySize64: 3823030272
  ###   WorkingSet: 53080064
  ###   WorkingSet64: 53080064

  ### CONTENTS OF "/proc/self/mountinfo":
  20 57 0:20 / /sys rw,nosuid,nodev,noexec,relatime shared:2 - sysfs sysfs rw,seclabel
  21 57 0:4 / /proc rw,nosuid,nodev,noexec,relatime shared:13 - proc proc rw
  22 57 0:6 / /dev rw,nosuid shared:9 - devtmpfs devtmpfs rw,seclabel,size=7905916k,nr_inodes=1976479,mode=755
  23 20 0:7 / /sys/kernel/security rw,nosuid,nodev,noexec,relatime shared:3 - securityfs securityfs rw
  24 22 0:21 / /dev/shm rw,nosuid,nodev shared:10 - tmpfs tmpfs rw,seclabel
  25 22 0:22 / /dev/pts rw,nosuid,noexec,relatime shared:11 - devpts devpts rw,seclabel,gid=5,mode=620,ptmxmode=000
  26 57 0:23 / /run rw,nosuid,nodev shared:12 - tmpfs tmpfs rw,seclabel,mode=755
  27 20 0:24 / /sys/fs/cgroup rw,nosuid,nodev,noexec,relatime shared:4 - cgroup2 cgroup2 rw,seclabel,nsdelegate
  28 20 0:25 / /sys/fs/pstore rw,nosuid,nodev,noexec,relatime shared:5 - pstore pstore rw,seclabel
  29 20 0:26 / /sys/fs/bpf rw,nosuid,nodev,noexec,relatime shared:6 - bpf bpf rw,mode=700
  52 20 0:27 / /sys/kernel/config rw,relatime shared:7 - configfs configfs rw
  57 0 253:0 / / rw,relatime shared:1 - ext4 /dev/mapper/luks-deadbeef-dead-beef-dead-deadbeef rw,seclabel
  30 20 0:19 / /sys/fs/selinux rw,relatime shared:8 - selinuxfs selinuxfs rw
  31 20 0:8 / /sys/kernel/debug rw,relatime shared:14 - debugfs debugfs rw,seclabel
  32 21 0:30 / /proc/sys/fs/binfmt_misc rw,relatime shared:15 - autofs systemd-1 rw,fd=37,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=16196
  33 57 0:31 / /tmp rw,nosuid,nodev shared:16 - tmpfs tmpfs rw,seclabel
  34 22 0:18 / /dev/mqueue rw,relatime shared:17 - mqueue mqueue rw,seclabel
  35 22 0:32 / /dev/hugepages rw,relatime shared:18 - hugetlbfs hugetlbfs rw,seclabel,pagesize=2M
  36 20 0:33 / /sys/fs/fuse/connections rw,relatime shared:19 - fusectl fusectl rw
  80 57 8:1 / /boot rw,relatime shared:39 - ext4 /dev/sda1 rw,seclabel
  83 57 253:1 / /home rw,relatime shared:41 - ext4 /dev/mapper/luks-deadbeef-dead-beef-dead-deadbeef rw,seclabel
  174 57 0:34 / /var/lib/nfs/rpc_pipefs rw,relatime shared:87 - rpc_pipefs sunrpc rw
  559 26 0:37 / /run/user/1000 rw,nosuid,nodev,relatime shared:308 - tmpfs tmpfs rw,seclabel,size=1584160k,mode=700,uid=1000,gid=1000
  349 559 0:35 / /run/user/1000/gvfs rw,nosuid,nodev,relatime shared:147 - fuse.gvfsd-fuse gvfsd-fuse rw,user_id=1000,group_id=1000
  ### CONTENTS OF "/proc/self/cgroup":
  0::/user.slice/user-1000.slice/user@1000.service/gnome-terminal-server.service
  ### CONTENTS OF "/proc/self/limits":
  Limit                     Soft Limit           Hard Limit           Units
  Max cpu time              unlimited            unlimited            seconds
  Max file size             unlimited            unlimited            bytes
  Max data size             unlimited            unlimited            bytes
  Max stack size            8388608              unlimited            bytes
  Max core file size        unlimited            unlimited            bytes
  Max resident set          unlimited            unlimited            bytes
  Max processes             61764                61764                processes
  Max open files            524288               524288               files
  Max locked memory         65536                65536                bytes
  Max address space         unlimited            unlimited            bytes
  Max file locks            unlimited            unlimited            locks
  Max pending signals       61764                61764                signals
  Max msgqueue size         819200               819200               bytes
  Max nice priority         0                    0
  Max realtime priority     0                    0
  Max realtime timeout      unlimited            unlimited            us
    Finished:    System.Runtime.InteropServices.RuntimeInformation.Tests
  === TEST EXECUTION SUMMARY ===
     System.Runtime.InteropServices.RuntimeInformation.Tests  Total: 8, Errors: 0, Failed: 0, Skipped: 0, Time: 0.199s
  ~/devel/dotnet/corefx/src/System.Runtime.InteropServices.RuntimeInformation/tests
  ----- end Mon 22 Jul 2019 10:07:44 PM EDT ----- exit code 0 ----------------------------------------------------------
  exit code 0 means Exited Successfully
```
</details>